tests/helpers.py

```diff
@@ -706,3 +707,122 @@ def get_worker_nodes():
+        sa_name (str): Serviceaccount name
+
+    Returns:
+        Pod: A Pod instance
+
+    Raises:
+        AssertionError: In case of any failure
+    """
+    if pod_dict_path:
+        pod_dict = pod_dict_path
+    else:
+        pod_dict = constants.FEDORA_DC_YAML
+    pod_data = templating.load_yaml_to_dict(pod_dict)
+    pod_name = create_unique_resource_name(
+        f'test-{interface_type.lower()}', 'pod'
+    )
+    pod_data['metadata']['name'] = pod_name
+    pod_data['metadata']['labels']['app'] = pod_name
+    pod_data['spec']['template']['metadata']['labels']['name'] = pod_name
+
+    pod_data['metadata']['namespace'] = namespace
+    if pvc_name:
+        pod_data['spec']['template']['spec']['volumes'][0]['persistentVolumeClaim']['claimName'] = pvc_name
+    if sa_name:
+        pod_data['spec']['template']['spec']['serviceAccountName'] = sa_name
+    ocs_obj = create_resource(wait=False, **pod_data)
+    logger.info(ocs_obj.name)
+    assert (ocp.OCP(kind='pod', namespace=namespace)).wait_for_resource(
+        condition=constants.STATUS_SUCCEEDED,
+        resource_name=pod_name + '-1-deploy ',
+        resource_count=0, timeout=180, sleep=3
+    )
+    dpod_list = pod.get_all_pods(namespace=namespace)
+    for dpod in dpod_list:
+        if '-1-deploy' not in dpod.name:
+            if pod_name in dpod.name:
+                return dpod
+
+
+def create_serviceaccount(namespace):
+    """
+    Create a Serviceaccount
+
+    Args:
+        namespace (str): The namespace for the serviceaccount creation
+
+    Returns:
+        OCS: An OCS instance for the service_account
+    """
+
+    service_account_data = templating.load_yaml_to_dict(
+        constants.SERVICE_ACCOUNT_YAML
+    )
+    service_account_data['metadata']['name'] = create_unique_resource_name(
+        'sa', 'serviceaccount'
+    )
+    service_account_data['metadata']['namespace'] = namespace
+
+    return create_resource(**service_account_data, wait=False)
+
+
+def add_scc_policy(sa_name, namespace):
+    """
+     Adding ServiceAccount to scc privilaged
+
+    Args:
+        sa_name (str): ServiceAccount name
+        namespace (str): The namespace for the scc_policy creation
+
+    """
+    kubeconfig = os.getenv('KUBECONFIG')
+    out = run_cmd(f"oc --kubeconfig {kubeconfig} adm policy add-scc-to-user privileged "
+                  f"system:serviceaccount:{namespace}:{sa_name}")
+    logger.info(out)
+
+
+def remove_scc_policy(sa_name, namespace):
+    """
+     Removing ServiceAccount from scc privilaged
+
+    Args:
+        sa_name (str): ServiceAccount name
+        namespace (str): The namespace for the scc_policy deletion
+
+    """
+    kubeconfig = os.getenv('KUBECONFIG')
+    out = run_cmd(f"oc --kubeconfig {kubeconfig} adm policy remove-scc-from-user privileged "
+                  f"system:serviceaccount:{namespace}:{sa_name}")
+    logger.info(out)
+
+
+def delete_deploymentconfig(pod_obj, namespace):
```

Comment:
not sure why we need this function. you can simply call OCP.delete(resouce_name=pod_obj.get_labels().get('name'))

Reply:
done